src/uwtools/tests/api/test_drivers.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-function-docstring,protected-access
+
```

Comment:
Given that the API driver modules no longer contain any significant code of their own and simply call code from other modules, it seemed to me sensible to combine their unit tests into a single module. This breaks the one-to-one mapping between library modules and test modules, but saves so much duplicated code that I thought the benefit outweighed that departure from convention. The per-driver test modules are removed.